LIBADD already fixed in master (0.7/0.8)

--- origsrc/libnotify-0.6.0/libnotify/Makefile.am	2010-11-17 12:19:14.377260700 -0600
+++ src/libnotify-0.6.0/libnotify/Makefile.am	2010-11-17 12:31:50.557276200 -0600
@@ -31,10 +31,10 @@ libnotify_la_SOURCES = \
 	notify-marshal.c
 
 libnotify_la_LIBADD = \
-	$(LINKS_LIBS)
+	$(PACKAGE_LIBS)
 
 libnotify_la_LDFLAGS = \
-	-version-info $(LT_CURRENT):$(LT_REVISION):$(LT_AGE)
+	-version-info $(LT_CURRENT):$(LT_REVISION):$(LT_AGE) -no-undefined
 
 notify-marshal.h: notify-marshal.list
 	$(GLIB_GENMARSHAL) --prefix=$(MARSHAL_PREFIX) notify-marshal.list \
